Cabal-syntax/src/Distribution/PackageDescription/Parsec.hs

```diff
@@ -88,7 +88,10 @@ parseGenericPackageDescription bs = do
       Just csv -> return (Just csv)
       Nothing ->
         parseFatalFailure zeroPos $
-          "Unsupported cabal-version " ++ prettyShow v ++ ". See https://github.com/haskell/cabal/issues/4899."
+          "Unsupported cabal-version specification of cabal format version in cabal-version field: "
```

Comment:
This seems repetitive. Perhaps:
```suggestion
          "Unsupported cabal format version in cabal-version field: "
```

Reply:
Thanks for the quick feedback! I agree, your version is better. Example:

```
Errors encountered when parsing cabal file ./records.cabal:

records.cabal:0:0: error:
Unsupported cabal format version in cabal-version field: 3.10.
Current cabal-version values are listed at https://cabal.readthedocs.io/en/stable/file-format-changelog.html.
```